x-pack/plugins/triggers_actions_ui/public/application/sections/alerts_table/alerts_table.tsx

```diff
@@ -18,13 +19,23 @@ const AlertsTable: React.FunctionComponent<AlertsTableProps> = (props: AlertsTab
     pageSize: props.pageSize,
   });
 
-  const [visibleColumns, setVisibleColumns] = useState(props.columns.map(({ id }) => id));
+  const alertsTableTypeRegistry = useKibana().services.alertsTableTypeRegistry;
+  if (!alertsTableTypeRegistry.has(props.ownerPluginId)) {
+    throw new Error(
+      'This plugin has no registered its alerts table parameters inside TriggersActionsUi'
+    );
+  }
+  const pluginCustomizations = alertsTableTypeRegistry.get(props.ownerPluginId);
```

Comment:
I'm having a hard time with the terminology here. The registry uses the term `type` and then we call it `pluginCustomizations` here - I think we need a better term for what's happening here. 

Maybe it's configuration? `alertsTableConfigurationRegistry` and then we can call this variable `alertsTableConfiguration`.

WDYT?

Reply:
I like it. I was not sure about this name I'll change the names to your suggested ones.